PROJECT

```diff
@@ -0,0 +1,3 @@
+version: "1"
```

Comment:
What file is this one? Is it part of the new revised tooling?

Reply:
Yes.  It was created by kubebuilder.